### PR TITLE
Run3-alca248X Get rid of some unnecessary include and a small improvement of a macro

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Get rid of some unnecessary include and a small improvement of a macro

#### PR validation:

Use the standard workflows in runTheMatrix

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalCalibAlgos/macros/CalibFitPlots.C [hunk 1]
```diff
@@ -4142,7 +4148,8 @@ void PlotHistCorrRatio(char* infile1,
       int npt(0);
       for (std::map<int, cfactors>::const_iterator itr = cfacs[ih].begin(); itr != cfacs[ih].end(); ++itr) {
         int ieta = (itr->second).ieta;
-        if ((ieta >= etamin) && (ieta <= etamax) && ((itr->second).depth == depth1)) {
+        bool seleta = (etaMin > 0) ? (std::abs(ieta) > etaMin) : true;
+        if ((ieta >= etamin) && (ieta <= etamax) && seleta && ((itr->second).depth == depth1)) {
```
Comment (perrotta) [inline Calibration/HcalCalibAlgos/macros/CalibFitPlots.C:4152]: The only difference with respect to the previous formulation is when `etamin>0`  and `etamin=ieta`, which was accepted with the previous formulation while it is rejected now.

In any case it does NOT correspond to what stated in the comment at the beginning of the file:
```C++
//      etaMin < 0 and etaMax > 0 will take ieta range from -etaMax to +etaMax;
//      etaMin > 0 will select ieta's where |ieta| is greater than etaMin
```

I think that what described in that comment should be coded as
```suggestion
        if ((ieta >= -etamax) && (ieta <= etamax) && seleta && ((itr->second).depth == depth1)) {
```

@bsunanda could you please check and confirm that this is exactly what you want (or fix that line if not)?

**Comment by bsunanda** [comment]
I tested the code and got what I desired and that is what is in the comment. Setting etaMin to a positive value overrides the test on etamin - this is all that is required. by setting etaMax=15 and etaMin=-1 we get only barrel. Setting etaMin=17 and etaMax to any value larger than 17 or even -1, i get only endcap. If I do not set any of etaMin or etaMax to any +ve value, I get the entire range covering both barrel and endcap

________________________________

**Comment by perrotta** [comment]
+1
- Ok, @bsunanda [claims](https://github.com/cms-sw/cmssw/pull/43389#issuecomment-1826733829) he checked all particular cases in the macro. Perhaps still some corner case is not considered (see https://github.com/cms-sw/cmssw/pull/43389#issuecomment-1826797593) but I imagine the macro is only fed by some appropriate input parameters. 
- I suspect it could have been coded in a more intuitive and simpler way, but it could work even as such...

**Pull request merged**